### DISCUSSION TITLE
Eclipse microprofile - 8 March

### DISCUSSION THREAD

**Discussion by planetf1** [odpi/egeria#7420] [Ideas]
As a followon to previous discussions, Emily Jiang (IBM) is available to speak to us about Eclipse microprofile & a comparison to spring on Wed 8 March in our regular timeslot.

I know we discussed yesterday using this session to plan out a poc in more detail, I think it's best we accomodate Emily's availability, especially as she's usually busy at this time.

Is that ok with everyone? Are you all available @lpalashevski @juergenhemelt @davidradl ?

**Comment by juergenhemelt** [comment]
For me it's ok 👍

**Comment by lpalashevski** [comment]
I agree. The proposal works for me as well.

**Comment by davidradl** [comment]
ok for me

**Comment by planetf1** [comment]
Thanks. Unfortunately our speaker is now not available.

We could do the following week, 15 March. There may also be an option for the Thursday

We'd originally intended discussion initial POC proposals on 8 March in any case, so I suggest we do that - as long as it works for you all? and I'll cancel the meeting for 1 march as we agreed.

**Reply by lpalashevski** [reply]
works for me.

**Reply by juergenhemelt** [reply]
ok for me
